src/serverless.js

```diff
@@ -157,11 +175,7 @@ class KnativeServing extends Component {
           metadata: {
             annotations
           },
-          spec: {
-            containers: [
-              imageConfig
-            ]
-          }
+          spec: templateSpec
```

Comment:
This change could have done in `serverless-knative` package. by monkey patching this `getManifest` method

and the changes in default is also can be handled in `serverless-knative` package.

I could have manged to patch only 1 package `serverless-knative` package.